### PR TITLE
[NEAT-226] Introduce last 👋

### PR DESCRIPTION
**Pull request by doctrino**
A tiny PR to add the new field `Last` to ensure we are aligned on what we are planning to do.

Splitting the refactoring work if using this new field into multiple PRs.

### CODE REVIEW

File: cognite/neat/rules/models/dms/_converter.py [hunk 1]
```diff
@@ -107,7 +107,8 @@ def as_information_architect_rules(
             metadata=metadata,
             properties=SheetList[InformationProperty](data=properties),
             classes=SheetList[InformationClass](data=classes),
-            reference=self.dms.reference and self.dms.reference.as_information_architect_rules(),  # type: ignore[arg-type]
+            last=self.dms.last.as_information_architect_rules() if self.dms.last else None,
```
Comment (nikokaoja) [inline cognite/neat/rules/models/dms/_converter.py:110]: I think we made a right choice of having added last, as we now have stack of three models. I imagine we can build rather strong UI lineage showing interconnection of these three.

File: cognite/neat/rules/models/dms/_rules.py [hunk 2]
```diff
@@ -322,15 +323,6 @@ def matching_version_and_space(cls, value: SheetList[DMSView], info: ValidationI
             warnings.warn(issues.dms.ViewModelSpaceNotMatchingWarning(different_space, metadata.space), stacklevel=2)
         return value
 
-    @field_validator("views")
```
Comment (nikokaoja) [inline cognite/neat/rules/models/dms/_rules.py:325]: Why is this removed?

**Review by nikokaoja — APPROVED** [review]

File: cognite/neat/rules/models/dms/_rules.py (hunk 2 above)
Comment (doctrino) [reply cognite/neat/rules/models/dms/_rules.py:325]: Bad merged, it is included in the validation above.
